addons/website_sale/tests/test_website_sale_pricelist.py

```diff
@@ -49,6 +49,7 @@ def setUpClass(cls):
         cls.country_be = cls.env.ref('base.be')
         cls.benelux = cls.env['res.country.group'].create({
             'name': 'BeNeLux',
+            'code': 'BE-NL-LU',
```

Comment:
It is not a required field, is it? :thinking: 

Why do you need to add it in tests?